src/main/java/online/carsharing/annotation/ValidRentalDates.java

```diff
@@ -12,7 +12,7 @@
 @Target({ ElementType.TYPE })
 @Retention(RetentionPolicy.RUNTIME)
 public @interface ValidRentalDates {
-    String message() default "Return date must be after rental date";
+    String message() default "Invalid dates! Please select future dates for rental";
```

Comment:
IMHO : Invalid dates! Please check dates: Keep in mind that RentalDate should start no less than today and ReturnDate should be at least 1 day after RentalDate